### PR TITLE
Add Gitlab docs

### PR DESCRIPTION
**Pull request by shrivara**
#299

### CODE REVIEW

File: sources/int_overview.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Delete will fail if there are any projects actively using the integration, so ma
 - GitHub
 - GitHub Enterprise
 - Bitbucket
+- Gitlab
 
 In addition, we are working on Bitbucket Server and GitLab integrations and will announce these in Q1 of 2016.
```
Comment (abhijitkini) [inline sources/int_overview.md]: Remove GitLab

File: sources/int_overview.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Delete will fail if there are any projects actively using the integration, so ma
 - GitHub
 - GitHub Enterprise
 - Bitbucket
+- Gitlab
```
Comment (abhijitkini) [inline sources/int_overview.md]: Please use the standard name of GitLab (G & L upper case) in all places.

File: sources/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ We run all your builds on **minions**, which are Docker-based containers. If you
 
 You should use Shippable for Continuous Integration if you fit the following profile -
 
-* Your source code is on GitHub, Bitbucket or GitHub Enterprise. (Support for Bitbucket Server and Gitlab will be announced in Q1 2016)
+* Your source code is on GitHub, Bitbucket, Gitlab or GitHub Enterprise. (Support for Bitbucket Server will be announced in Q1 2016)
```
Comment (abhijitkini) [inline sources/index.md]: Please use the standard name of GitLab (G & L upper case) in all places.

File: sources/index.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ page_keywords: containers, lxc, Docker, Continuous Integration, Continuous Deplo
 
 Shippable was founded so that software development teams could accelerate their release cycles by automating the build, test and deployment of applications. Our Continuous Delivery pipeline takes your code from source control to production, without the need for configuration management/IT automation tools or infrastructure code.
 
-We integrate with the tools and technologies you use today and will need tomorrow, like GitHub, Bitbucket and on-prem repositories, offer seamless deployment to private clouds as well as leading cloud providers including Amazon and Google, and support all application architectures from enterprise grade multi-tier to containerized micro services.
+We integrate with the tools and technologies you use today and will need tomorrow, like GitHub, Bitbucket, Gitlab and on-prem repositories, offer seamless deployment to private clouds as well as leading cloud providers including Amazon and Google, and support all application architectures from enterprise grade multi-tier to containerized micro services.
```
Comment (abhijitkini) [inline sources/index.md]: Please use the standard name of GitLab (G & L upper case) in all places.

**Comment by abhijitkini** [comment]
@shrivara Great job in updating the docs. Few things before it can be merged:

- Please use the standard name of GitLab (G & L upper case) in all places.
- I have one comment in the PR where you need to remove GitLab from a sentence.

**Comment by shrivara** [comment]
@abhijitkini I've made the changes, please check.

File: sources/ht_using_gitlab.md [hunk 5]
```diff
@@ -1,271 +1,27 @@
...
-different provider or even hosting the repository server on your own. In
-both cases, the easiest way to make your code available to Shippable is
-to set up a mirror of your repository with either of the supported
-services.
+Shippable supports running CI builds for GitLab. To setup this, you'll first
+need sign up to Shippable using a Github or Bitbucket account. Following this,
+you can follow the steps mentioned [here](int_scm.md#gitlab) to add GitLab as
+an Account Integration and sync your Subscriptions. Once done, you can continue
+on to enable any project and setup a `shippable.yml` file to get your builds
+running.
```
Comment (abhijitkini) [inline sources/ht_using_gitlab.md]: I would reword few sentences as:

- To set this up, you'll first need to sign into Shippable using a GitHub or BitBucket. Then follow the steps mentioned...
- Once complete, you can enable a project/repository and setup

File: sources/ht_using_gitlab.md [hunk 6]
```diff
@@ -1,271 +1,27 @@
...
-granted that no more than 5 users will have access to them. It makes it
-especially appealing to host mirrors then, as you only need to associate
-two users (the account you use with Shippable and an extra account for
-your repository server) with all the mirrors.
+## Builds
 
-Please write down the Git url of your newly created mirror. For the sake
-of convenience, we will use the following urls throughout this guide:
+Builds are automatically triggered every time you push code or open a new
+merge request.
```
Comment (abhijitkini) [inline sources/ht_using_gitlab.md]: every time you commit code or open a new pull request.

File: sources/ht_using_gitlab.md [hunk 7]
```diff
@@ -1,271 +1,27 @@
...
-`hg-git` documentation for more details.
-
-One particular problem connected to this issue is that Mercurial
-bookmarks are bound to the local repository and are not pushed. This
-means that the bookmarks are invisible to 'central' repository that has
-the hook in place (and sends the changes to Git). The only solution here
-is to clone (i.e. fork) Mercurial repositories and keep separate Git
-mirrors for them, instead of creating branches.
+Shippable supports running builds for you merge requests, so you can merge
+confidently without breaking your app.
```
Comment (abhijitkini) [inline sources/ht_using_gitlab.md]: Shippable triggers builds for all commits & merge requests, hence you can commit/merge code confidently without breaking your app.

File: sources/int_overview.md [hunk 8]
```diff
@@ -43,8 +43,9 @@ Delete will fail if there are any projects actively using the integration, so ma
 - GitHub
 - GitHub Enterprise
 - Bitbucket
+- GitLab
 
-In addition, we are working on Bitbucket Server and GitLab integrations and will announce these in Q1 of 2016.
+In addition, we are working on Bitbucket Server integration and will announce this in Q1 of 2016.
```
Comment (abhijitkini) [inline sources/int_overview.md]: Q1 has already passed. Use Q2 of 2016.

File: sources/index.md [hunk 9]
```diff
@@ -27,7 +27,7 @@ We run all your builds on **minions**, which are Docker-based containers. If you
 
 You should use Shippable for Continuous Integration if you fit the following profile -
 
-* Your source code is on GitHub, Bitbucket or GitHub Enterprise. (Support for Bitbucket Server and Gitlab will be announced in Q1 2016)
+* Your source code is on GitHub, Bitbucket, GitLab or GitHub Enterprise. (Support for Bitbucket Server will be announced in Q1 2016)
 * You want to avoid the time sink of setting up and managing your own CI system and would rather spend the time writing features for your product
```
Comment (abhijitkini) [inline sources/index.md]: Change Q1 to Q2

**Comment by abhijitkini** [comment]
Some more feedback included above.

Important: When 3rd party services are being mentioned, do ensure the exact syntax and words are used. For example. GitLab (which you fixed throughout), GitHub (and not Github), etc.

**Comment by shrivara** [comment]
@abhijitkini, updated the Gitlab docs. Also, changed Github to GitHub everywhere and removed some trailing spaces.
